backend/lib/plugins/error-handler.js

```diff
@@ -5,6 +5,8 @@ const Sentry = require("@sentry/node");
 // https://github.com/fastify/fastify-sensible/blob/5eec059b2a77579f75bafc91e12f60fb90b6dab5/index.js#L46
 function defaultErrorHandler(error, request, reply) {
   if (
+    reply &&
+    reply.raw &&
     reply.raw.statusCode === 500 &&
```

Comment:
what you could also do to make this shorter, not sure if it works in Node 12: `reply?.raw?.statusCode` but I think this currently only works in the frontend

Reply:
It only works in Node 14: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining